dist/calendar/CalendarMonth.js

```diff
@@ -65,7 +60,33 @@ var CalendarMonth = _react2['default'].createClass({
     highlightedRange: _react2['default'].PropTypes.object,
     onMonthChange: _react2['default'].PropTypes.func,
     onYearChange: _react2['default'].PropTypes.func,
-    value: _utilsCustomPropTypes2['default'].momentOrMomentRange
+    value: _utilsCustomPropTypes2['default'].momentOrMomentRange,
+    locale: _react2['default'].PropTypes.string
+  },
+
+  _setupLocale: function _setupLocale(locale) {
+    this.moment = (0, _moment2['default'])();
+
+    this.moment.locale(locale);
+
+    var lang = this.moment.localeData(locale);
+
+    this.WEEKDAYS = _immutable2['default'].List(lang._weekdays).zip(_immutable2['default'].List(lang._weekdaysShort));
```

Comment:
@ashmigelski actually the problem is that there's an issue with the locale definition in the `ru` locale. When locale is `ru`, check `lang.hasOwnProperty('_months')`, for russian it will return `false`, which means that the months are not in the locale data so it's being loaded from the defaults in the prototype (english). That's why in the dropdown of the months there are russian months